pytorch_lightning/callbacks/early_stopping.py

```diff
@@ -134,26 +134,32 @@ def load_state_dict(self, state_dict):
         self.best_score = state_dict['best_score']
         self.patience = state_dict['patience']
 
-    def on_sanity_check_end(self, trainer, pl_module):
```

Comment:
this one is not needed anymore?

Reply:
nope. early stopping now gets its info from on_validation_epoch_end

Reply:
since sanity check is the special case where validation loop has 2 batches